test/functional/admin_controller_test.rb

```diff
@@ -92,7 +92,7 @@ def teardown
  	assert_equal 'Reset your password', email.subject
  	assert_equal [user.email], email.to
 
-    assert_equal "#{user.name} should receive an email with instructions on how to reset their password. If they do not, please double check that they are using the email they registered with.", flash[:notice] 
+    assert_equal "#{user.name} should receive an email with instructions on how to reset their password. If they do not, please double check that they are using the email they registered with.", flash[:notice]
```

Comment:
Remove all the changes done in this file, as you have to create a new test for the mailer.

Reply:
Oh, I understand. I thought that I had to include the mailer test in the test of the method already created. So, do you want that I created a a new empty test case for mailer? In this file, right?

Reply:
OHHHH, You just want the mailer test!  but, Don't you want to test that the controller send the mail??

Reply:
Perhaps Gaurav is thinking that a separate test would be good so that each test is narrow and tests only one thing. In any case if they are clearly titled a testing different things that's a good thing. Does that help? Sorry I am not completely up to date on this but I think Gaurav is not available.

Reply:
@jywarren you are right. Actually, as we just started on this project, I thought we should create a temporary method which would call the mailer. And, then we would create a test for testing that temporary function, so this way we would not be changing any existing code. But, I admit that after going through the work in this PR, I think we can start this mailer. I really didn't expected while creating the issue that it would be understood so well but @ValentinaTironi has done the connecting work too. Awesome :tada: :100:!